docs/quickstart.rst

```diff
@@ -90,8 +90,9 @@ backed up and that the ``prune`` command is keeping and deleting the correct bac
     export BORG_PASSCOMMAND='pass show backup'
 
     # some helpers and error handling:
-    function info  () { echo -e "\n"`date` $@"\n" >&2; }
-    trap "echo `date` Backup interrupted >&2; exit 2" SIGINT SIGTERM
+    info() { printf "\n%s %s\n\n" "$( date )" "$*" >&2; }
+    # OR: info() { echo; echo "$( date ) $*" >&2; echo; }
```

Comment:
It works both and both POSIX-compatible. What do you consider to look "better"/"nicer"?